### PR TITLE
product: change viewer limits to 10k

### PR DESCRIPTION
**Pull request by gioelecerati**
**What does this pull request do? Explain your changes. (required)**

- Change the default max concurrent viewers to 10k
- Handle sandbox plan concurrent viewers limit on the admin frontend

**Specific updates (required)**

**How did you test each of these updates (required)**

**Does this pull request close any open issues?**

**Screenshots (optional)**

**Checklist**

- [ ] I have read the **CONTRIBUTING** document.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.

### CODE REVIEW

File: packages/www/pages/app/admin/user/[id].tsx [hunk 1]
```diff
@@ -51,7 +51,8 @@ const ID = () => {
   const [enableModal, setEnableModal] = useState(false);
   const [updateViewerLimitModal, setUpdateViewerLimitModal] = useState(false);
   const [newViewerLimit, setNewViewerLimit] = useState(
-    userInfo?.viewerLimit || 50000,
+    userInfo?.viewerLimit ||
+      (userInfo?.stripeProductId === "prod_O9XuIjn7EqYRVW" ? 30 : 10_000),
```
Comment (leszko) [inline packages/www/pages/app/admin/user/[id].tsx:55]: What is this `prod_O9XuIjn7EqYRVW`

Comment (gioelecerati) [reply packages/www/pages/app/admin/user/[id].tsx:55]: It's the sandbox plan. Unluckily I have to hardcode it until we have products in a proper `product` table when we finally do billing v2

**Review by leszko — APPROVED** [review]

File: packages/www/pages/app/admin/user/[id].tsx [hunk 2]
```diff
@@ -194,7 +195,10 @@ const ID = () => {
                   </Box>
                   <Box sx={{ m: "0.4em" }}>Viewer limit</Box>
                   <Box sx={{ m: "0.4em" }}>
-                    {userInfo.viewerLimit ? userInfo.viewerLimit : 50000}
+                    {userInfo.viewerLimit ??
+                      (userInfo?.stripeProductId === "prod_O9XuIjn7EqYRVW"
+                        ? 30
```
Comment (thomshutt) [inline packages/www/pages/app/admin/user/[id].tsx:200]: should this be 30000?

Comment (gioelecerati) [reply packages/www/pages/app/admin/user/[id].tsx:200]: This is just the value displayed on the admin page for `Sandbox` users, which is 30!
The value are these:
- Sandbox `30`
- Enterprise `20_000`
- All the others `10_000`

Comment (thomshutt) [reply packages/www/pages/app/admin/user/[id].tsx:200]: Ah right, of course!

**Review by thomshutt — APPROVED** [review]

**Pull request merged**